keyboards/exclusive/e7v1se/keymaps/mac/keymap.c

```diff
@@ -0,0 +1,35 @@
+#include QMK_KEYBOARD_H
```

Comment:
Technically, since the only difference between this keymap and the default is the swapped Alt/GUI keys, the latter could simply have the `AG_TOGG` keycode mapped instead.

Reply:
Thanks for the suggestion. Wasn't aware of this feature and had search around to get some information about this.
In the end I decided to keep the full mac keymap to make it easier for anyone to change or extend this keymap.
If the `AG_TOGG` is the better and standard way please provide me with more info or an example how to use it.

Reply:
Yeah, the `AG_TOGG` is the better way to handle that.   It's part of bootmagic, and specifically "magic", so it's present on every board.   Also, these settings are saved to eeprom, so they will be retained even when unplugging the keyboard.  

You can see a full list of the magic keycodes here:
https://docs.qmk.fm/#/feature_bootmagic?id=keycodes

Reply:
I read the docs about this toggle before and currently there are two keymaps using it in this repo.
My assumption is that if you know how to operate QMK as a user and you know if you are a Mac/Linux/Windows user you probably want the mods in a fixed spot.
I can add the `AG_TOGG` to the keymaps on the second layer. Can't hurt, but I need to test it myself first.
Priority now is to get this commit in the repo for my fellow groupbuy members.

Reply:
The `AG_TOGG` toggle is added to layer 1 of both keymaps and I've tested it works as expected.
This means a user can decide which base keymap to flash and have to option to toggle the mods if needed.